### ISSUE TITLE
new rule, omit_receiving

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1425]
https://github.com/SAP/styleguides/blob/master/clean-abap/CleanABAP.md#omit-receiving

I have no clue why this is not implemented yet 🤷 

example,
```abap
    upload_pack(
      EXPORTING
        io_client       = lo_client
        iv_url          = iv_url
        iv_deepen_level = iv_deepen_level
        it_hashes       = lt_hashes
      RECEIVING
        rt_objects      = et_objects ).
```

**Comment by larshp** [comment]
should not report for:

dynamic calls,
```abap
          CALL METHOD ('ZCL_ABAPGIT_OBJECT_SICF')=>read_tadir
            EXPORTING
              iv_pgmid    = iv_pgmid
              iv_obj_name = iv_obj_name
            RECEIVING
              rs_tadir    = rs_tadir.
```

calls with exceptions,
```abap
    cl_ci_checkvariant=>get_ref(
      EXPORTING
        p_user                   = ''
        p_name                   = iv_variant
      RECEIVING
        p_ref                    = ro_variant
      EXCEPTIONS
        chkv_not_exists          = 1
        missing_parameter        = 2
        OTHERS                   = 3 ).
```

**Comment by larshp** [comment]
https://syntax.abaplint.org/?filter=method#/expression/MethodCallParam

**Comment by larshp** [comment]
I'm starting this now

**Comment by larshp** [comment]
done in 2.56.0
